### ISSUE TITLE
More decimals

### ISSUE THREAD

**Issue by housemobile** [mobnetic/BitcoinChecker#272]
Lots of trades fluctuate on the 4th or 5th decimal yet the app only shows up to 3 decimals so it never looks like it changes. 

Would be nice to be able to set the # of decimals shown

**Comment by ungaro59** [comment]
Yes, me too,  i want to select the number of decimals to view for each trades.  Thank you in advance.
